src/components/pages/doc/sidebar/item/item.jsx

```diff
@@ -40,19 +46,28 @@ const Item = ({ title, slug = null, isStandalone = null, items = null, currentSl
         type={slug ? undefined : 'button'}
         to={slug ? externalSlug || docSlug : undefined}
         target={externalSlug ? '_blank' : '_self'}
+        aria-label={ariaLabel}
         onClick={handleClick}
       >
-        <span className="leading-snug">{title}</span>
-        <ChevronRight
+        {ariaLabel && <span className="sr-only">{ariaLabel}</span>}
+        <span className="leading-snug" aria-hidden={!!ariaLabel}>
+          {title}
+        </span>
+        <span
           className={clsx(
-            'mx-2 mt-[5px] shrink-0 transition-[transform,color] duration-200',
+            'block h-4 w-4 transition-[transform,background-color] duration-200',
             currentSlug === slug
-              ? 'text-black-new dark:text-white'
-              : 'text-gray-new-40 group-hover:text-black-new dark:text-gray-new-90 dark:group-hover:text-white',
-
+              ? 'bg-black-new dark:bg-white'
+              : 'bg-gray-new-40 group-hover:bg-black-new dark:bg-gray-new-90 dark:group-hover:bg-white',
             items?.length ? 'block' : 'hidden',
             isOpen ? 'rotate-90' : 'rotate-0'
           )}
+          style={{
+            WebkitMaskImage: "url('/images/chevron-right.svg')",
+            WebkitMaskSize: '6px 10px',
+            WebkitMaskRepeat: 'no-repeat',
+            WebkitMaskPosition: 'center',
```

Comment:
Very interesting solution, can you tell more about how does it work?

Reply:
As we need to color the SVGs, one of the easy ways to do it is using a CSS mask.

Reply:
https://codepen.io/noahblon/post/coloring-svgs-in-css-background-images#css-masks-1